examples/webgl_loader_multiple.html

```diff
@@ -0,0 +1,314 @@
+
+					// disposing all loaded meshes, materials and textures in the currently loaded model
+					currentModel.traverse( function ( node ) {
+
+						if ( ! node.isMesh ) return;
+
+						node.geometry.dispose();
+						const materials = Array.isArray( node.material ) ? node.material : [ node.material ];
+						materials.forEach( ( material ) => {
+
+							for ( const key of Object.keys( material ) ) {
+
+								if ( material[ key ] && material[ key ].isTexture ) {
+
+									material[ key ].dispose();
+
+								}
+
+							}
+
+							material.dispose();
+
+						} );
+
+					} );
+
+				}
+
+				currentModel = subscene;
+				scene.add( currentModel );
+
+				// enable shadows
+				currentModel.traverse( function ( child ) {
+
+					if ( child.isMesh ) {
+
+						child.castShadow = true;
+						child.receiveShadow = true;
+
+					}
+
+				} );
+
+				fitCameraToObjects( camera, controls, [ currentModel ], 1.25 );
+
+				render();
+
+			}
+
+    		// Adapted from https://discourse.threejs.org/t/camera-zoom-to-fit-object/936/24
+			// Slower but better implementation that takes bones and exact vertex positions into account: https://github.com/google/model-viewer/blob/04e900c5027de8c5306fe1fe9627707f42811b05/packages/model-viewer/src/three-components/ModelScene.ts#L321
+			function fitCameraToObjects( camera, controls, objects, fitOffset = 1.5 ) {
+
+				const size = new THREE.Vector3();
+				const center = new THREE.Vector3();
+				const box = new THREE.Box3();
+
+				box.makeEmpty();
+				for ( const object of objects )
+					box.expandByObject( object );
+
+				box.getSize( size );
+				box.getCenter( center );
+
+				const maxSize = Math.max( size.x, size.y, size.z );
+				const fitHeightDistance = maxSize / ( 2 * Math.atan( Math.PI * camera.fov / 360 ) );
+				const fitWidthDistance = fitHeightDistance / camera.aspect;
+				const distance = fitOffset * Math.max( fitHeightDistance, fitWidthDistance );
+
+				const direction = controls.target.clone()
+					.sub( camera.position )
+					.normalize()
+					.multiplyScalar( distance );
+
+				controls.maxDistance = distance * 10;
+				controls.target.copy( center );
+
+				camera.near = distance / 100;
+				camera.far = distance * 100;
+				camera.updateProjectionMatrix();
+
+				camera.position.copy( controls.target ).sub( direction );
+
+				controls.update();
+
+			}
+
+			function loadFile( file ) {
+
+				const filename = file.name;
+				const extension = filename.split( '.' ).pop().toLowerCase();
+
+				// To support .gltf, folder drop support would need to be added and the GLTFLoader paths set correctly. Omitted here for briefness.
+				if ( extension == 'glb' ) {
+
+					var reader = new FileReader();
+
+					reader.addEventListener( 'load', function ( event ) {
+
+						const loader = new GLTFLoader();
+						loader.setKTX2Loader( ktx2Loader );
+						loader.setMeshoptDecoder( MeshoptDecoder );
+						loader.parse( reader.result, '', gltf => addToScene( gltf.scene ) );
```

Comment:
To clarify, you don't think we need to support animations in this example, correct?

Reply:
I'd prefer if animations would be supported – I see it as an example to consolidate ideas around "loading files and viewing them" into one place. Added that again